packages/fx-core/resource/backup-config-change-logs.md

```diff
@@ -0,0 +1,24 @@
+# Congratulations! You are now switching to another tenant or subscription. Configuration files for last provision/local debug are saved successfully.
+
+You are now using a Microsoft 365 account or an Azure subscription (applies only when provisioning in a remote environment of a project requiring cloud resources) that is different from what you used previously in the selected environment for provision or local debugging. We have backed up the configuration files which are used for or generated from previous provision or local debug. For more information, you can check this [doc](https://aka.ms).
+
+## Why backup
+Configuration files will be overwritten by Teams Toolkit when provisioning in an already-provisioned environment but with different account or Azure subscription or local debugging again with another Microsoft 365 account. We will back up those files so that you could continue using the resources created before easily when you decide to switch back to the accounts or the subscription that you selected before. Otherwise, new resources will be created, and you have to delete the old resources manully to avoid costs.
+
```

Comment:
Another reason we do backup besides "recover" is to help developers locate the resources we created e.g. the Teams app id so they could delete them manually, Probably we can mention that as well.